IntoleranceSimulator.R

```diff
@@ -1,3 +1,51 @@
 # LT 4/05/2022
 
-# Simulation module
\ No newline at end of file
+
+# Intolerance simulation module
+
+# Use a statistical model to generate correlated random intolerance
+# scores (o/e) on alphafold protein structures
+
+
+## required packages
+require(bio3d)
+
+# for multivariate Gaussian RNG
+require(MASS)
+
+# protein structure file
+# hard coded for quick prototyping
+
+protfile = 'structures/AF-P68133-F1-model_v2.pdb'
+
+pdb = read.pdb(protfile)
+
+# calculate residues pairwise distance matrix
+distmat = dm(pdb)
+
+# is upper triangular with empty diagonal: add diagonal, fill lower triangle, convert to matrix

```

Comment:
Maybe 'this is an upper triangular with empty diagonal'?

Reply:
Will modify